dafni_cli/tests/commands/test_helpers.py

```diff
@@ -89,6 +90,66 @@ def test_resource_not_found(self, mock_click, mock_get_latest_dataset_metadata):
         self.assertEqual(err.exception.code, 1)
 
 
+class TestCliSelectDatasetFiles(TestCase):
+    """Test class to test cli_select_dataset_files"""
+
+    def setUp(self) -> None:
+        super().setUp()
+
+        self.file_names = ["file1.csv", "file2.zip", "file3.csv"]
+        self.dataset_metadata = MagicMock(
+            files=[DataFile(name=file_name, size=0) for file_name in self.file_names]
+        )
+
+    def test_all_optionals_none_returns_whole_list(self):
+        """Tests that the entire list of files is returned when both file_regex
```

Comment:
`file_regex`? Is this comment outdated or am I not understanding what it's referring to?

Reply:
Yeah just outdated